kv-components/vue/KvButton.vue

```diff
@@ -99,6 +106,16 @@ export default {
 		return {};
 	},
 	computed: {
+		loadingColor() {
```

Comment:
This is what I inferred from the figma, where each button variant has a specific loading color? Not sure if this is correct, also ghost button doesnt have a loading icon specified in the figma, so I just assumed it was brand color.

Reply:
Seems like a good path to take. Maybe we can call this out to design when they review in storybook.